blackbox/requirements.txt

```diff
@@ -14,9 +14,8 @@ minio>=5.0.0
 dnslib
 
 # Documentation
-sphinx>=4.6
+crate-docs-theme==0.35.0.dev3
```

Comment:
Using 0.35.0.dev3 only for evaluation and preview purposes before running a GA release.

Reply:
[0.35.0](https://pypi.org/project/crate-docs-theme/0.35.0/) has been released.